### DISCUSSION TITLE
Regarding updation in domain.

### DISCUSSION THREAD

**Discussion by harxh-v** [community/community#55807] [GitHub Education] [answered]
Dear, while registering to Students/Campus account, Students are seeing another email domain (@gpranchi.edu.in) which is no longer our school domain. Somehow they applied using school ID. The domain mentioned above) was handed over to NIC (a government agency) for the handling of website.
Further, our requirement of email and Google Workspace wasn't accepted by the Government agency, so after approval from the government the domain of our college was updated to (gpranchi.org) and the email domain to (@gpranchi.org)

Further requesting to update the domain list.

**Comment by samus-aran** [comment, marked as answer]
Hi @iiHR5  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
